Makefile

```diff
@@ -175,6 +175,13 @@ deploy: update-mgr-env manifests kustomize ## Deploy controller to the K8s clust
 	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+helm-chart: update-mgr-env manifests kustomize ## Create helm chart
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG} && $(KUSTOMIZE) edit set nameprefix ${MANAGER_NAME_PREFIX}
+	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
+	$(KUSTOMIZE) build config/default > charts/templates/template.yaml
```

Comment:
should we use `kustomize build --enable-helm` to build helm charts?

Reply:
that's cool, how much effort is required to edit final charts created by kustomize from any prior experience?

Reply:
not sure about it, i just came to know that we can also build helm from kustomize tool https://tech.aabouzaid.com/2020/09/3-ways-to-customize-off-the-shelf-helm-charts-with-kustomize-kubernetes.html#method-1-kustomize-plugins https://github.com/kubernetes-sigs/kustomize/blob/master/examples/chart.md#helm-related-flags